### ISSUE TITLE
Does the optimizer reset between tasks?

### ISSUE THREAD

**Issue by tianylin98** [Qznan/SpanKL#5]
If my understanding of the codes is correct, the optimizer doesn't really resets when shifting from one task to the next... Is that right? So generally the learning rate gradually decays throughout the process. I used to assume that in continual learning, when training a new model, we only possess knowledge of the model parameters, but not the training status of previous model (e.g., optimizer states and last learning rate).

I'm new to this area. Is this a common setting in continual learning (or continual NER)?

**Comment by Qznan** [comment]
We reset the optimizer and also the lr_schedule explicitly between tasks, e.g., Line 612-613 in [train_clner.py](https://github.com/Qznan/SpanKL/blob/main/train_clner.py).

**Comment by tianylin98** [comment]
Thank you for the reply. That resolves my concern.
